hms-app/src/actions/CategoryActions.tsx

```diff
@@ -1,9 +1,9 @@
 const core_url = process.env.REACT_APP_CORE_URL
 
 export const addCategory = (props: {
+  contextID: string
```

Comment:
I think still naming it hackathonId is good, then you can be sure without other information of what value to pass into here

Reply:
the problem is that conextID sometimes is a category id and sometimes a hackathon id and have to name both the same

Reply:
but for this function you only ever use it as a hackathonID